src/proj.rs

```diff
@@ -202,9 +205,11 @@ impl ProjBuilder {
     ///
     /// # Safety
     /// This method contains unsafe code.
+    #[doc(cfg(feature = "network"))]
```

Comment:
Unlike the exercise we just did in geojson, I'm doing the `doc(cfg` on all configurations. 

Do you know if there was a reason to go through the extra ceremony that we were going through there?

Reply:
AFAIK because it's a nightly-only feature so enabling it by default can break the build?

Reply:
oh right, of course. That's not gonna work. 

I'll follow the pattern you had in geojson.

Reply:
Hmm... I attempted to do this, but TBH I cannot figure out the appropriate invocation to verify the results.

```
$ cargo +nightly doc --features network --cfg docsrs 
error: Found argument '--cfg' which wasn't expected, or isn't valid in this context

USAGE:
    cargo doc --features <FEATURES>...

For more information try --help
```

```
$ cargo +nightly doc --features network -- --cfg docsrs
error: Found argument '--cfg' which wasn't expected, or isn't valid in this context

USAGE:
    cargo doc --features <FEATURES>...

For more information try --help
```

```
$ cargo +nightly rustdoc --features network --cfg docsrs
error: Found argument '--cfg' which wasn't expected, or isn't valid in this context

USAGE:
    cargo rustdoc --features <FEATURES>...

For more information try --help
```


Maybe this one is most promising:

```
cargo +nightly rustdoc --features network -- --cfg docsrs
 Documenting proj v0.20.3 (/Users/mkirk/src/georust/proj)
error[E0658]: `#[doc(cfg)]` is experimental
   --> src/proj.rs:208:24
    |
208 |     #[cfg_attr(docsrs, doc(cfg(feature = "network")))]
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #43781 <https://github.com/rust-lang/rust/issues/43781> for more information
    = help: add `#![feature(doc_cfg)]` to the crate attributes to enable

error: Compilation failed, aborting rustdoc

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
error: Could not document `proj`.
```

Reply:
Oh yeah I tried it earlier too and then decided to live a little. If it works for Tokio it's almost certainly OK, and this will precede a new geojson release so if it _does_ break on docs.rs it's less of a big deal and I'll take responsibility for puzzling it out. Yes, I care about release quality but rustdoc is too mysterious.